evm/src/libraries/ExecutorMessages.sol

```diff
@@ -63,4 +64,12 @@ library ExecutorMessages {
     function makeCCTPv1Request(uint32 sourceDomain, uint64 nonce) internal pure returns (bytes memory) {
         return abi.encodePacked(REQ_CCTP_V1, sourceDomain, nonce);
     }
+
+    /// @notice Encodes a version 2 CCTP request payload.
+    /// @param sourceDomain The source chain for the CCTP transfer.
+    /// @param nonce The nonce of the CCTP transfer.
```

Comment:
From our convo, just leaving a comment here that nonce is no longer returned and is [set to 0 on-chain](https://github.com/circlefin/evm-cctp-contracts/blob/master/src/messages/v2/MessageV2.sol#L93). Judging by this [testnet tx](https://iris-api-sandbox.circle.com/v2/messages/1?transactionHash=0x80ef5bf2cae555d827a5f3b745f9f1d107bd3a7ce786d07f87a6e68ce535e7d4) API response, it also isn't u64 anymore, but maybe a hash of something?